src/CuFourierFlows.jl

```diff
@@ -1,4 +1,4 @@
-using CuArrays
+using .CuArrays
```

Comment:
@glwagner, what does this `.` do?

Reply:
I don't know, you put it there..... and **I** asked you what does it do and you never replied!